src/docs/product/sentry-basics/search/index.mdx

```diff
@@ -10,6 +10,16 @@ description: "Learn more about Search, which is available in several Sentry feat
 
 Search is available on several features throughout [sentry.io](https://sentry.io), such as **Issues**, **Discover** and **Dashboards**.
 
+## Filters
```

Comment:
The challenging content for users to understand on this page is the syntax, so i don't want that get buried below the fold. I think we should move this section to the end (just before Learn More). 
```suggestion
## Page Filters
```